api/src/services/healthService.js

```diff
@@ -106,27 +106,29 @@ async function liveness(req, res) {
 async function readiness(req, res) {
   let errors = [];
   
-  try {
-    await getClient();
-  } catch (err) {
-    errors.push(`Unable to connect to database:${err.message}`);
-  }
-
-  if (!cacheService.initialized()) {
-    errors.push('Cache service is not initialized yet');
-  }
-
-  if (!cereNetworkService.initialized()) {
-    errors.push('Cere network service is not initialized yet');
-  }
-
-  if (!ethNetworkService.initialized()) {
-    errors.push('Ethereum network service is not initialized yet');
-  }
+    try {
+      const client = await getClient();
+      client.end();
+    } catch (error) {
+      console.error(error);
```

Comment:
Why do we need console.log?

Reply:
We need to see in the logs which exact database error happened.

Reply:
When we deploy a pod the `readiness` API is not publicly available it database error happens this is why we need to see it in the logs.